### PR TITLE
Add types

### PR DESCRIPTION
**Pull request by mrzmmr**
Adds types and tests copying what Roang-zero1 did for [unist-util-find-all-after](https://github.com/syntax-tree/unist-util-find-all-after/pull/7).

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "dtslint/dtslint.json",
+  "rules": {
+    "semicolon": false,
+    "whitespace": false,
+    "strict-export-declare-modifiers": false
+  }
+}
```
Comment (ChristianMurphy) [inline tslint.json]:
```suggestion
}

```

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "lib": ["es2015"],
+    "strict": true,
+    "baseUrl": ".",
+    "paths": {
+      "unist-util-find-all-before": ["index.d.ts"]
+    }
+  }
+}
```
Comment (ChristianMurphy) [inline tsconfig.json]:
```suggestion
}

```

**Review by ChristianMurphy — APPROVED** [review]
Thanks @mrzmmr! :bow:

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
Thanks @mrzmmr, released in 3.0.0!

**Pull request merged**